e2e/coverage-handlebars/greet.hbs

```diff
@@ -5,9 +5,9 @@
  * LICENSE file in the root directory of this source tree.
 }}
 <p>Good
-{{#if am}}
-  morning
-{{else}}
-  evening
-{{/if}}
-{{name}}!</p>
+  {{#if am}}
+    morning
+  {{else}}
+    evening
+  {{/if}}
+  {{name}}!</p>
```

Comment:
Looks like a bug in Prettier, it removed the final new line.

Reply:
this is the failing test as well btw, changed the coverage report. Can add an ignore comment so we can land this PR?